### PR TITLE
Large office prototype computer room AC efficiency

### PR DESCRIPTION
**Pull request by yunyangye**
This PR updates the computer room AC efficiency for large office prototype model. This implementation affects the prototype models for large office. Additional information regarding the implementation are provided in Section B2.8 in the [Energy Savings Analysis: ANSI/ASHRAE/IES Standard 90.1-2019 report](https://www.energycodes.gov/sites/default/files/2021-07/20210407_Standard_90.1-2019_Determination_TSD.pdf).

### CODE REVIEW

File: lib/openstudio-standards/prototypes/common/objects/Prototype.CoilCoolingWaterToAirHeatPumpEquationFit.rb [hunk 1]
```diff
@@ -61,4 +61,52 @@ def create_coil_cooling_water_to_air_heat_pump_equation_fit(model,
...
+        end
+        cop = crac_minimum_scop / crac_sensible_cool_ratio
+        cop = cop.round(2)
+        OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.CoilCoolingWaterToAirHeatPumpEquationFit', "For #{coil_cooling_water_to_air_heat_pump.name}: #{cooling_type} #{heating_type} #{sub_category} Capacity = #{capacity_kbtu_per_hr.round}kBtu/hr; SCOP = #{crac_minimum_scop}")
+      end
+    else
+      cop = nil
+    end
+    return cop
+  end
```
Comment (yunyangye) [inline lib/openstudio-standards/prototypes/common/objects/Prototype.CoilCoolingWaterToAirHeatPumpEquationFit.rb:111]: Calculate CRAC efficiency based on SCOP and sensible cooling ratio.

File: lib/openstudio-standards/prototypes/common/objects/Prototype.Model.rb [hunk 2]
```diff
@@ -1587,6 +1587,22 @@ def model_apply_prototype_hvac_efficiency_adjustments(model)
...
+          cop = coil_cooling_water_to_air_heat_pump_standard_minimum_cop_crac(coil_cooling_water_to_air_heat_pump)
+          if cop.nil?
+            OpenStudio.logFree(OpenStudio::Info, 'openstudio.model.Model', "COP for #{coil_cooling_water_to_air_heat_pump.name} is not changed")
+          else
+            coil_cooling_water_to_air_heat_pump.setRatedCoolingCoefficientofPerformance(cop)
+          end
+        end
+      end
+    end
+
```
Comment (yunyangye) [inline lib/openstudio-standards/prototypes/common/objects/Prototype.Model.rb:1605]: Reset CRAC efficiency for large office prototype model.

File: lib/openstudio-standards/prototypes/common/objects/Prototype.CoilCoolingWaterToAirHeatPumpEquationFit.rb [hunk 3]
```diff
@@ -61,4 +61,51 @@ def create_coil_cooling_water_to_air_heat_pump_equation_fit(model,
...
+        else
+          OpenStudio.logFree(OpenStudio::Error, 'openstudio.standards.CoilCoolingWaterToAirHeatPumpEquationFit', 'Failed to get autosized sensible cool capacity')
+        end
+        cop = crac_minimum_scop / crac_sensible_heat_ratio
+        cop = cop.round(2)
+        OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.CoilCoolingWaterToAirHeatPumpEquationFit', "For #{coil_cooling_water_to_air_heat_pump.name}: #{cooling_type} #{heating_type} #{sub_category} Capacity = #{capacity_kbtu_per_hr.round}kBtu/hr; SCOP = #{crac_minimum_scop}")
+      end
+    end
+    return cop
+  end
```
Comment (yunyangye) [inline lib/openstudio-standards/prototypes/common/objects/Prototype.CoilCoolingWaterToAirHeatPumpEquationFit.rb:110]: Calculate CRAC efficiency based on SCOP and sensible heat ratio.

**Review by mdahlhausen — CHANGES_REQUESTED** [review]
This method sets efficiency for CoilCoolingWaterToAirHeatPumpEquationFit objects modding the `coil_cooling_water_to_air_heat_pump_standard_minimum_cop` method.  It looks up efficiency in new data tables in the `unitary_acs` data, but the method it is based on uses the `water_source_heat_pumps` data.  Instead of creating an entirely new method to set efficiency, adjust the existing method to take an optional argument and include the new data there.

coil_cooling_water_to_air_heat_pump

**Comment by lymereJ** [comment]
`coil_cooling_water_to_air_heat_pump_standard_minimum_cop_crac` was moved to be part of `coil_cooling_water_to_air_heat_pump_standard_minimum_cop`.

**Review by mdahlhausen — APPROVED** [review]

**Pull request merged**